### PR TITLE
Make flowId query param optional for GET microservices endpoint

### PR DESCRIPTION
**Pull request by sergerad**

### CODE REVIEW

File: src/services/microservices-service.js [hunk 1]
```diff
@@ -39,12 +39,17 @@ const MicroserviceExtraHostManager = require('../data/managers/microservice-extr
...
 
 async function listMicroservicesEndPoint (flowId, user, isCLI, transaction) {
-  if (!isCLI) {
+  if (!isCLI && flowId) {
     await _validateFlow(flowId, user, isCLI, transaction)
   }
 
-  const where = isCLI ? { delete: false } : { flowId: flowId, delete: false }
-  const microservices = await MicroserviceManager.findAllExcludeFields(where, transaction)
+  let microservices = null
```
Comment (Pixcell) [inline src/services/microservices-service.js]: Should be `let microservices = []`.  But it does not really matter

**Review by Pixcell — APPROVED** [review]

File: src/services/microservices-service.js [hunk 2]
```diff
@@ -39,12 +39,17 @@ const MicroserviceExtraHostManager = require('../data/managers/microservice-extr
...
   }
 
-  const where = isCLI ? { delete: false } : { flowId: flowId, delete: false }
-  const microservices = await MicroserviceManager.findAllExcludeFields(where, transaction)
+  let microservices = []
+  if (flowId) {
+    const where = isCLI ? { delete: false } : { flowId: flowId, delete: false }
+    microservices = await MicroserviceManager.findAllExcludeFields(where, transaction)
+  } else {
+    microservices = await MicroserviceManager.findAll({ flowId: { [Op.ne]: null }, delete: false }, transaction)
```
Comment (baghbidi) [inline src/services/microservices-service.js]: Need to add user id to `where` object, I think.

**Review by baghbidi — CHANGES_REQUESTED** [review]

File: src/services/microservices-service.js (hunk 2 above)
Comment (Pixcell) [reply src/services/microservices-service.js]: Oh yes, good catch!

**Review by Pixcell — CHANGES_REQUESTED** [review]
Same a Saeid. Need to add userId to the microservice query when querying all

**Review by baghbidi — APPROVED** [review]

**Pull request merged**